airbyte-integrations/connectors/source-file/airbyte_protocol

```diff
@@ -0,0 +1 @@
+../../bases/base-python/airbyte_protocol
```

Comment:
what is this file achieving?

Reply:
This symlink is copied from the template:
https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/python-source/airbyte_protocol

It is there so when you open your IDE in one of the connector-templates/python-source root directory, references to airbyte_protocol library are properly resolved and highlighting, code navigation etc are working nicely. 

So it's mostly for DX purposes